lib/pages/child/panel_page.dart

```diff
@@ -1,27 +1,113 @@
+import 'dart:developer';
 import 'package:flutter/material.dart';
+import 'package:fokus/utils/app_locales.dart';
+import 'package:fokus/utils/theme_config.dart';
 import 'package:fokus/widgets/app_navigation_bar.dart';
 import 'package:fokus/widgets/child_wallet.dart';
+import 'package:fokus/widgets/plan_widget.dart';
 
 class ChildPanelPage extends StatefulWidget {
-	@override
-	_ChildPanelPageState createState() => new _ChildPanelPageState();
+  @override
+  _ChildPanelPageState createState() => new _ChildPanelPageState();
 }
 
 class _ChildPanelPageState extends State<ChildPanelPage> {
-	@override
-	Widget build(BuildContext context) {
-		return Scaffold(
-			body: Column(
-				crossAxisAlignment: CrossAxisAlignment.start,
-				children: [
-					ChildCustomHeader(),
-					Container(
-						padding: EdgeInsets.all(8.0),
-						child: Text('Dzisiejsze plany', textAlign: TextAlign.left, style: Theme.of(context).textTheme.headline2)
-					)
-				]
-			),
-			bottomNavigationBar: AppNavigationBar.childPage(currentIndex: 0)
-		);
-	}
+  // styling vars
+  final Color futurePlansColor = Colors.amber;
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
```

Comment:
Try to break functions this long into logical smaller parts. Here the current, todays, and button for example